packages/truffle-box/lib/utils/unbox.js

```diff
@@ -81,13 +76,45 @@ function fetchRepository(url, dir) {
   });
 }
 
-function copyTempIntoDestination(tmpDir, destination) {
-  return new Promise(function(accept, reject) {
-    fs.copy(tmpDir, destination, function(err) {
-      if (err) return reject(err);
-      accept();
-    });
-  });
+async function copyTempIntoDestination(tmpDir, destination) {
+    const currentContent = fs.readdirSync(destination);
+      if (currentContent.length > 1) {
+        const tmpContent = fs.readdirSync(tmpDir);
+        for (let file of tmpContent) {
+           if (currentContent.includes(file)) {
+             var overwriting = [
+  		{
```

Comment:
Indentation...also I personally would prefer that this is written like
```
let variableName = [
  {
    type: 'blah,
    otherType: 'otherBlah',
  }
]
```
To me this is much more readable.